src/app/components/MediaLoader/Placeholder/index.styles.tsx

```diff
@@ -17,6 +17,13 @@ const styles = {
         '> button': {
           backgroundColor: palette.POSTBOX,
         },
+        '.experimentButtonFocus': {
```

Comment:
Are we planning to run this as an experiment? Or can we use the word `transcript` instead?